salt/metalk8s/repo/init.sls

```diff
@@ -1,2 +1,2 @@
 include:
-  - .offline
+  - .redhat
```

Comment:
This is a bit odd. Shouldn't we dispatch on host OS here and include the 'correct' file accordingly?